lib/services/registered_addons.pm

```diff
@@ -84,7 +90,7 @@ sub check_suseconnect_cmd {
     my $status_out = script_output("SUSEConnect --status-text", 120);
     diag "$status_out";
     for (my $i = 0; $i < @addons; $i = $i + 1) {
-        next if ($addons[$i] =~ /^SLE(S|D|_HPC)$/);
+        next if ($addons[$i] =~ /^SLE(S|D|_HPC)$|^sle-module-packagehub-subpackages$/);
```

Comment:
Here need show the softfail.

Reply:
This is not softfail. As before it blocked by register status check. 
'sle-module-packagehub-subpackages' is not an addon.

Reply:
Ok.